canonical-web-and-design/ubuntu.com
# Fix eslint errors for desktop statistics

## Done

Fix eslint errors for static/js/src/desktopStatistics.js

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/statistics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `./run exec ./node_modules/.bin/eslint static/js/src/desktopStatistics.js`
- Check that there are no linting errors/warnings
- Check that /desktop/statistics looks and behaves the same as on ubuntu.com


## Issue / Card

Fixes #6889
